frontend-react/src/components/header/OrgDropdown.tsx

```diff
@@ -1,6 +1,6 @@
 import { Dropdown } from "@trussworks/react-uswds";
 import { CSSProperties, useState } from "react";
-import { useHistory } from "react-router";
```

Comment:
i think we can now remove `react-router` as a dep? Could be done as a separate pr

Reply:
Went ahead and took care of it here 👍🏼